src/transaction/posting.rs

```diff
@@ -69,10 +70,19 @@ impl<'a> Posting<'a> {
         self.info.price.as_ref().map(|(t, p)| (*t, p))
     }
 
+    /// Returns the lot attributes (if present)
+    #[must_use]
+    pub fn lot_attributes(&self) -> Option<&LotAttributes<'a>> {
```

Comment:
keep it private for now

```suggestion
    fn lot_attributes(&self) -> Option<&LotAttributes<'a>> {
```